src/server/services/wms/qgswmsrenderer.cpp

```diff
@@ -546,9 +547,34 @@ namespace QgsWms
       handlePrintErrors( layout.get() );
     }
 
+    // remove temeporary themes
+    // remove temeporary highlight layers
```

Comment:
Ping @3nids Hum... The spell checker is not working on C++ comments? Is is really useful to have the spell checker on the CI.